pubspec.yaml

```diff
@@ -30,6 +30,10 @@ dependencies:
       ref: account-url-fix
   http: ^0.13.1
   clipboard: ^0.1.3
+  web3dart: ^2.1.3
+  wallet_core:
+    git:
+      url: https://github.com/wal33d006/wallet_core.git
```

Comment:
I see this is a forked version of wallet_core, what is the difference between this and the original wallet_core?

Reply:
The library was outdated, not migrated to null-safety. I migrated it and create a PR in the original repo [here](https://github.com/fuseio/wallet_core/pull/64). Till the owner migrates it to null-safety, or my PR gets approved, I am using my own forked version.

Reply:
makes sense, thanks! definetely lets stick with the forked version for now :)